### DISCUSSION TITLE
Fundme contract

### DISCUSSION THREAD

**Discussion by 01sw01** [Cyfrin/foundry-full-course-cu#2932] [Q&A]
If i click Fund button in remi, it is giving me this error message;

......................
[vm]from: 0x5B3...eddC4to: FundMe.fund() 0x332...D4B6Dvalue: 10000000000000000 weidata: 0xb60...d4288logs: 0hash: 0xbca...2c513
transact to FundMe.fund errored: Error occurred: revert.

revert
	The transaction has been reverted to the initial state.
Note: The called function should be payable if you send value and the value you send should be less than your current balance.
If the transaction failed for not having enough gas, try increasing the gas limit gently.
........................

**Comment by EngrPips** [comment]
Hello @01sw01, Please share your code. Showing a full screenshot of your remix workspace will help us help you as well.

**Comment by KMean** [comment]
Hi @01sw01... would be great if you post at least your code otherwise is tricky to understand where the problem is. Said that looking at the revert make sure your fund function is payable. 

```solidity
function fund() public payable {
        require(msg.value.getConversionRate() >= MINIMUM_USD, "You need to spend more ETH!");
        // require(PriceConverter.getConversionRate(msg.value) >= MINIMUM_USD, "You need to spend more ETH!");
        addressToAmountFunded[msg.sender] += msg.value;
        funders.push(msg.sender);
    }
```
